cmd/ui/main.go

```diff
@@ -40,12 +40,14 @@ const (
 	defThingsAuthURL     = "localhost:8181"
 	defAuthURL           = "localhost:8189"
 	defThingsAuthTimeout = "1s"
+	defAuthToken         = "123"
```

Comment:
There will not be such a thing, as it will not be used in single-user mode. This is maybe your development helper, but not needed to be set-up globlay later.